workloads/templates/workload-pvcscale-script-cm.yml.j2

```diff
@@ -0,0 +1,367 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scale-ci-workload-script
+data:
+  run.sh: |
+    #!/bin/sh
+    # pbench Configuration
```

Comment:
As a best practice we should have `set -eo pipefail` up here.

Reply:
we have in all scripts `pbench configuration` and that part is same , easy to fix it here, but I would like the most to have this part in single file valid for all tests

Reply:
We could refactor at some point the pbench configuration part of the run script.  For now I think it's simple enough to remain a boiler plate code but I'm open to ideas.  As mentioned I think all scripts should run with the above statement though.

Reply:
added `set -eo pipefail`